docs/modules/ROOT/pages/index.adoc

```diff
@@ -498,13 +504,127 @@ In each of the above sections, the following changes MUST be applied:
+
+.Basic trust triangle
+image::trust-model/basic-trust-triangle.drawio.svg[Basic trust triangle,width=400,height=130,align="center"]
+
+The three roles depicted can each be performed by a human, organization, machine, or some combination thereof. A _credential holder_ establishes a relationship with a _credential issuer._ If the issuer trusts the credential holder, it will then issue a digital credential which makes statements about the credential holder and is signed by the issuer.
+
+Later, the _credential holder_ can then present this credential to a _verifier_ (also often known as a “relying party”). If the verifier has an existing trust relationship with the credential issuer, then the verifier can choose to trust the credential and the statements made within it.
+
+This pattern can be repeated if there is not a direct trust relationship between the _verifier_ and _issuer._ The issuer itself might have a credential that is issued by another _issuer_ that is known to the _verifier_ as shown in the following diagram:
+
+.Transitive trust triangle
+image::trust-model/transitive-trust-triangle.drawio.svg[Transitive trust triangle,width=570,height=220,align="center"]
+
+In this scenario, _issuer 2_ is playing a dual role as credential holder and credential issuer. The _verifier_ does not have a direct relationship with the issuer of the credentials that were presented (_issuer 2_). However, it can inspect _issuer 2’s_ credentials and find that they were issued by _issuer 1,_ with whom it does have a direct relationship. Based on the nature of that relationship, it may choose to extend *transitive trust* to _issuer 2_ and thus to the credential that _issuer 2_ issued.
+
+Web browsers provide a well-known example of transitive trust. Browsers have direct relationships with relatively few root trust anchors. Those anchors, known as _root certificate authorities,_ in turn issue credentials to _certificate authorities_ who then issue credentials to individual web site operators who then sign the content presented to a browser. This pattern may be repeated with multiple layers of intermediate certificate authorities. The web browser evaluates this entire chain of credentials when deciding whether to present a web site as trusted or not.
+
+=== Trust scenarios in identity assertion
+
+The `signature` in an *<<_identity_assertion,identity assertion>>* could be considered as a new credential documenting the relationship between _<<_named_actor,named actor>>_ and _<<C2PA asset>>,_ which will be referred to in this section as an _asset-specific credential._
+
+IMPORTANT: “Content Credential” is a trademarked term that refers to _<<_c2pa_manifest,C2PA Manifests>>_ and MUST NOT be used in reference to *<<_identity_assertion,identity assertion>>* signatures.
+
+A _<<C2PA Manifest Consumer>>_ SHOULD maintain a list of trust relationships that is is prepared to accept when interpreting any *<<_identity_assertion,identity assertion>>.*
```

Comment:
No!

A C2PA Manifest Consumer knows NOTHING about this assertion.

A CAWG-aware Manifest Consumer, however, could do this...